### PR TITLE
Create Alive.js

### PR DESCRIPTION
**Pull request by OnlyBard**

### CODE REVIEW

**Comment by TOXIC-DEVIL** [comment]
Please change file name to lowercase letters.

**Review by toxic-devil — CHANGES_REQUESTED** [review]
Please change file name to lowercase letters.

File: commands/alive.js
Comment (TOXIC-DEVIL) [inline commands/alive.js:1]: Thanks ~

**Review by toxic-devil — APPROVED** [review]
Thanks ~

**Pull request merged**